feedback.md

```diff
@@ -0,0 +1,34 @@
+# Feedback
+
+See [rubric](https://github.com/ga-wdi-exercises/project2/blob/master/evaluation.md)
+
+## Technical Requirements
+
+The app contains 2 models, with at least 1 association. At least 1 model is full CRUD. You've got 3 tables in your database and are doing some cool stuff user auth and limiting controller action to authorized users.
+
+Evaluation: *Performing*
+
+## Creativity and Interface
+
+The app is styled and has an interface of value to the end user: it is not just a nav bar and an index page. This is the area that your app could most improve. It would have been awesome if you had done a little more to make the interface your own instead of relying so heavily on Tunr. Don't be afraid to try new things, that's what creativity is about!
+
```

Comment:
Not surprising, because the only thing I did here was resize the text entry field.   ;)